File: WebShopAppBackend/webshop/src/main/java/org/vs/dto/ProizvodSpecifikacijaZahtjev.java
package org.vs.dto;

import lombok.*;

import jakarta.validation.constraints.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProizvodSpecifikacijaZahtjev {

	@Size(min = 3, max = 50)
	private String vrijednost;

	@NotNull
	private Integer proizvodId;

	@NotNull
	private Integer specifikacijaId;

}

File: WebShopAppFrontend/web-shop-app/src/app/model/proizvodSpecifikacijaZahtjev.ts
export class ProizvodSpecifikacijaZahtjev {

  vrijednost: string;
  proizvodId: number;
  specifikacijaId: number;

  constructor(
    vrijednost: string,
    proizvodId: number,
    specifikacijaId: number
  ) {
    this.vrijednost = vrijednost;
    this.proizvodId = proizvodId;
    this.specifikacijaId = specifikacijaId;
  }

}
